src/command_modules/azure-cli-subscription/azure/cli/command_modules/subscription/_params.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.command_modules.subscription._completers import get_offer_type_completion_list
+
+
+# pylint: disable=line-too-long
+def load_arguments(self, _):
+    with self.argument_context('subscriptiondefinition show') as c:
+        c.argument('subscription_definition_name', options_list=['--name', '-n'], required=False, help='Name of the subscription definition to show.')
```

Comment:
You don't need to call out "...to show".  That way you only need to apply this alias once, with a scope of `account definition`.

Reply:
Is there any reason why by default, "show" takes a "subscription_definition_name", but "create" takes a "name" parameter despite the swagger/Python SDK taking a "subscription_definition_name" parameter?

Reply:
Fixed.